widget/templates/_manifest.json

```diff
@@ -3,9 +3,9 @@
   "label": "<%= widgetTitle %>",
   "2D": <%= is2d %>,
   "3D": <%= is3d %>,
-  "platform": "HTML",
-  "version": "0.0.1",
-  "wabVersion": "1.4",
+  "platform": "<%= platform %>",
+  "version": "<%= version %>",
```

Comment:
So I thought the same, except all the new 3d widgets have 2.0beta as version as well. But I agree, it makes sense to leave version as 0.0.1